Can't npm install on Alpine Linux 3.19 because of spdlog

<!-- ⚠️⚠️ Do Not Delete This! bug_report_template ⚠️⚠️ -->
<!-- Please read our Rules of Conduct: https://opensource.microsoft.com/codeofconduct/ -->
<!-- 🕮 Read our guide about submitting issues: https://github.com/microsoft/vscode/wiki/Submitting-Bugs-and-Suggestions -->
<!-- 🔎 Search existing issues to avoid creating duplicates. -->
<!-- 🧪 Test using the latest Insiders build to see if your issue has already been fixed: https://code.visualstudio.com/insiders/ -->
<!-- 💡 Instead of creating your report here, use 'Report Issue' from the 'Help' menu in VS Code to pre-fill useful information. -->
<!-- 🔧 Launch with `code --disable-extensions` to check. -->
Does this issue occur when all extensions are disabled?: Yes

<!-- 🪓 If you answered No above, use 'Help: Start Extension Bisect' from Command Palette to try to identify the cause. -->
<!-- 📣 Issues caused by an extension need to be reported directly to the extension publisher. The 'Help > Report Issue' dialog can assist with this. -->
- VS Code Version: 1.85.1
- OS Version: Alpine 3.19

Steps to Reproduce:

1. `npm install`

```
10.55 npm ERR! ../deps/spdlog/include/spdlog/details/os-inl.h:241:19: error: aggregate 'spdlog::details::os::filesize(FILE*)::stat64 st' has incomplete type and cannot be defined
10.55 npm ERR!   241 |     struct stat64 st;
10.55 npm ERR!       |                   ^~
10.55 npm ERR! ../deps/spdlog/include/spdlog/details/os-inl.h:242:11: error: '::fstat64' has not been declared; did you mean 'stat64'?
10.55 npm ERR!   242 |     if (::fstat64(fd, &st) == 0)
10.55 npm ERR!       |           ^~~~~~~
10.55 npm ERR!       |           stat64
```

More info and fix in new `@vscode/spdlog` version : https://github.com/microsoft/node-spdlog/pull/42

--- a/package.json
+++ b/package.json
@@ -73,7 +73,7 @@
     "@vscode/policy-watcher": "^1.1.4",
     "@vscode/proxy-agent": "^0.18.2",
     "@vscode/ripgrep": "^1.15.9",
-    "@vscode/spdlog": "^0.13.12",
+    "@vscode/spdlog": "^0.14.0",
     "@vscode/sqlite3": "5.1.6-vscode",
     "@vscode/sudo-prompt": "9.3.1",
     "@vscode/vscode-languagedetection": "1.0.21",

--- a/remote/package.json
+++ b/remote/package.json
@@ -9,7 +9,7 @@
     "@vscode/iconv-lite-umd": "0.7.0",
     "@vscode/proxy-agent": "^0.18.2",
     "@vscode/ripgrep": "^1.15.9",
-    "@vscode/spdlog": "^0.13.12",
+    "@vscode/spdlog": "^0.14.0",
     "@vscode/vscode-languagedetection": "1.0.21",
     "@vscode/windows-process-tree": "^0.5.0",
     "@vscode/windows-registry": "^1.1.0",

